scripts/vim-patch.sh

```diff
@@ -256,7 +256,8 @@ list_vim_patches() {
   for vim_commit in ${vim_commits}; do
     local is_missing
     local vim_tag
-    vim_tag="$(cd "${VIM_SOURCE_DIR}" && git describe --tags --exact-match "${vim_commit}" 2>/dev/null)"
+    # This fails for untagged commits (e.g., runtime file updates) so mask the return status
+    vim_tag="$(cd "${VIM_SOURCE_DIR}" && git describe --tags --exact-match "${vim_commit}" 2>/dev/null)" || true
```

Comment:
Moving it implicitly ignores the error due to local succeeding, regardless of how the value for the variable is generated.  Explicitly ignoring the exit status seems clearer.